feat: Add a feature to change country display format in form (#95)
odolbeau/phone-number-bundle: README.md

By default the `country_choices` is empty, which means all countries are included, as is `preferred_country_choices`.
The option `country_placeholder` can be specified to create a placeholder option on above the whole list.

> The option `country_display_type` can be specified to change the country dropdown label format. There is three formats availables :

Reviewer: ```suggestion
The option `country_display_type` can be specified to change the country dropdown label format. There are two formats available :
```

You suggest only 2 formatting for the country name, isn't it? 🤔

Author: In the first place, i've planned to have 3 formats. But the third, only the country code (eg. +33) does not work because a region code can be used by many regionCode.
